packages/core/src/codewhisperer/views/lineAnnotationController.ts

```diff
@@ -92,10 +98,15 @@ class PressTabState implements AnnotationState {
     id = PressTabState.id
 
     suppressWhileRunning = false
+
     text = () => 'CodeWhisperer Tip 1/3: Press [TAB] to accept ([ESC] to exit)'
 
-    nextState(changeSource: AnnotationChangeSource, force: boolean): AnnotationState | undefined {
-        return new AutotriggerState().nextState(changeSource, force)
+    updateState(changeSource: AnnotationChangeSource, force: boolean): AnnotationState | undefined {
+        return new AutotriggerState().updateState(changeSource, force)
+    }
+
+    isNextState(state: AnnotationState | undefined): boolean {
+        return state instanceof ManualtriggerState
```

Comment:
Note that `case_1` and `case_1a` are both `case_1`, therefore their "nextState" is always `case_2` (ManualTriggerState)